ophyd/controls/device.py

```diff
@@ -233,7 +240,77 @@ def __new__(cls, name, bases, clsdict):
         return clsobj
 
 
-class OphydDevice(OphydObject, metaclass=ComponentMeta):
+
+# These stub 'Interface' classes are the apex of the mro heirarchy for
+# their respective methods. They make it safe to use multiple inheritance
+# with the base classes given in any order.
```

Comment:
It makes it *more* forgiving but not universally forgiving, depending on what you are doing.